.gitignore

```diff
@@ -23,4 +23,5 @@ README_PARAMETERS.txt
 build/
 cython_cygno.c
 reco*.txt
-*debug.txt
\ No newline at end of file
+*debug.txt
+configFile_LNGS_low_gain.txt
```

Comment:
I would remove line 27 as it is specific to your use. For the new commits you can git add specific files.